template/readme/local-development.md

```diff
@@ -74,17 +92,22 @@ Then follow the Quick Start Guide in [Drupal VM's README](https://github.com/gee
     # `php5-xsl` to `extra_packages`.
     extra_packages:
       - unzip
-      - php5-xsl
+    # - php5-xsl
```

Comment:
The xsl package was required to run PHPUnit tests, I think, associated with BLT from a while back. I don't remember if it's still required or not.